src/app/components/login/components/reset-forgotten-password.jsx

```diff
@@ -5,19 +5,22 @@ import {
   CardHeader,
   CardContent,
   makeStyles,
+  Typography,
 } from '@material-ui/core';
 import VpnKeyIcon from '@material-ui/icons/VpnKey';
 import { func, string } from 'prop-types';
 import { useFormik } from 'formik';
 import * as yup from 'yup';
 import { useSnackbar } from 'notistack';
 
+import { getAngularService } from 'services/angular-react-helper';
```

Comment:
This should be after the "eventTrack" import. This section is for imports that are relative to the file, but this import is in our project.